PayPalNativeCheckout/build.gradle

```diff
@@ -37,7 +37,7 @@ dependencies {
     implementation deps.appCompat
     implementation project(':PayPalDataCollector')
 
-    implementation('com.paypal.checkout:android-sdk:0.8.2') {
+    implementation('com.paypal.checkout:android-sdk:0.8.6') {
```

Comment:
these are pretty minor changes, we have been building upon a feature branch just adding small things while we wait for GA